### PR TITLE
[BUG] getIsRootPageId must handle 0 

### PR DESCRIPTION
**Pull request by thomashohn**
The method getRootPageId should always return an int

Fixes:  #1077

### CODE REVIEW

**Comment by thomashohn** [comment]
I'll look into that - or we can look at it Thursday..

**Comment by timohund** [comment]
Yes :)

File: Classes/Domain/Index/Queue/RecordMonitor/Helper/RootPageResolver.php [hunk 1]
```diff
@@ -129,7 +134,7 @@ public function getRootPageId($pageId = 0, $forceFallback = false)
     {
         /** @var Rootline $rootLine */
         $rootLine = GeneralUtility::makeInstance(Rootline::class);
-        $rootPageId = intval($pageId) ? intval($pageId) : $GLOBALS['TSFE']->id;
+        $rootPageId = intval($pageId) ? intval($pageId) : intval($GLOBALS['TSFE']->id);
```
Comment (irnnr) [inline Classes/Domain/Index/Queue/RecordMonitor/Helper/RootPageResolver.php]: I think you can just use tertiary short notation here `?:`

**Comment by timohund** [comment]
@thomashohn Thanks for the pull request!

**Pull request merged**